README.md

```diff
@@ -1,5 +1,7 @@
 # Vue Sample Applications for Okta
 
+> Note: You're currently looking at sample applications that support Vue 3. If you are looking for samples for Vue 2, please checkout the [main branch](https://github.com/okta/samples-js-vue).
```

Comment:
After this PR gets merged into the `vue-3` branch, we can add a similar note in the main branch.